src/include/binder/ddl/bound_alter_info.h

```diff
@@ -49,15 +50,17 @@ struct BoundExtraRenameTableInfo : public BoundExtraAlterInfo {
 struct BoundExtraAddPropertyInfo : public BoundExtraAlterInfo {
     std::string propertyName;
     common::LogicalType dataType;
-    std::shared_ptr<Expression> defaultValue;
+    std::unique_ptr<parser::ParsedExpression> defaultValue;
```

Comment:
I wanted to use binding information originally, but setting the serialization and deserialization for all the expression classes did not go very smooth so I scrapped that. Also, duckdb uses parsed expressions rather than bound expressions, which probably has some reason behind their choice to do that